bucket/imagemagick.json

```diff
@@ -19,7 +22,7 @@
         "conjure.exe",
         "convert.exe",
         "dcraw.exe",
-        "ffmpeg.exe",
+        ["ffmpeg.exe", "iffmpeg"],
```

Comment:
The ffmpeg executive shipped with Imagemagick is rather dated. Aliasing the shim allows us to install the [ffmpeg](https://github.com/lukesampson/scoop/blob/master/bucket/ffmpeg.json) package alongside Imagemagick. Is there a better way?